website/content/en/highlights/2021-11-12-event-trottle-transform.md

```diff
@@ -0,0 +1,52 @@
+---
+date: "2021-11-16"
+title: "Event `throttle` transform released"
+description: "A guide to using the new `throttle` transform"
+authors: ["barieom"]
+pr_numbers: []
+release: "0.18.0"
+hide_on_release_notes: false
+badges:
+  type: new feature
+---
+
+# A new `throttle` transform 
+
+
+We've released a new `throttle` transform that provides a user the ability to throttle the throughput of specific event streams.
+
+Large spikes in data volume can frequently overwhelm a service, which is especially common in log data. Previously, users lacked the necessary tooling to control throughput, such as setting quota for users and user groups utilizing Vector, which not only can cause spike in costs, but also negatively impact downstream services due to the increased load. 
+
+The [`throttle`][] transform enables you to rate limit specific subsets of your event stream to limit load on downstream services or enforce quotas on users. You can utilize the `throttle` transform to enforce rate limits on number of events and exclude events based on a [VRL condition][] to avoid dropping critical logs. 
+
+To ensure that each bucket's throughput averages out to the `threshold` per `window`, rate limiting spreads load across the configured `window`. The rate limiter will allow up to threshold number of events through and drop any further events for that particular bucket when the rate limiter is at capacity. 
+
+A rate limiter is created with a maximum number of cells equal to the `threshold`, with cells replenishing at a rate of `window` divided by `threshold`. To paint a better example, a `window` of 60 with a `threshold` of 10 replenishes a cell every 6 seconds and allows a burst of up to 10 events. 
+
+For example,
```

Comment:
As mentioned this example is pretty trivial (and is exactly what's on the documentation page). I'm fine with this as I wrote it originally :D but if we have the time @binarylogic's suggestion of graphing the effects of the throttle transform might be nice.